### DISCUSSION TITLE
About 1G memory version

### DISCUSSION THREAD

**Discussion by Boos4721** [Boos4721/openwrt#57] [Announcements]
![iShot2021-12-28 23 50 32](https://user-images.githubusercontent.com/43845654/147583647-b9d24bcf-2a4f-422f-8e7a-790505175d06.png)
**DO IT, GUYS.**

**Comment by Boos4721** [comment]


**Reply by felipejfc** [reply]
the 1gb memory profile option only appears for me in the master branch, there's no option in the stable one

**Reply by Boos4721** [reply]
> the 1gb memory profile option only appears for me in the master branch, there's no option in the stable one

yus.

**Reply by felipejfc** [reply]
but in master branch build my ax3600 keeps restarting when I enable wifi

**Reply by Boos4721** [reply]
> but in master branch build my ax3600 keeps restarting when I enable wifi

ya, Im waiting for new backport mac80211.

**Comment by davnozdu** [comment]
What model chip you install ?
AX3600 have EM6HE16EWAKG-10IH 933MHz 1866Mbps/pin SDRAM - DDR3L
You install same model with more memory or another DDR3L (from motherboard or RAM slot) ?

**Reply by wutaijieing** [reply]
d9stq and d9swb

**Comment by wutaijieing** [comment]
1G版本固件我测试了，结果是需要用ttl救砖
自编译和你编译的都是一样的问题：1.无法保存nvram，导致无法切换回rootfs12分区。2.eth0变成了lan eth1变成了wan，eth2/3未使用。3.无线wifi均无法启动。4.无法通过sysupgrade命令恢复到正常的固件。
[AX3600_ttl_2021-12-31_5_39_44.log](https://github.com/Boos4721/openwrt/files/7794824/AX3600_ttl_2021-12-31_5_39_44.log)
。

**Reply by wutaijieing** [reply]
对了，29号前的正常版本5Gwifi貌似有点问题，iphone13无法连接，（其他设备连接正常）设置为wpa和wpa2都一样，改成AC模式才成功连接

**Reply by felipejfc** [reply]
> 对了，29号前的正常版本5Gwifi貌似有点问题，iphone13无法连接，（其他设备连接正常）设置为wpa和wpa2都一样，改成AC模式才成功连接

I also have the same problem with my iPhone 12 Pro. I can only get stable connection if I set 5Ghz AC 80mhz

**Reply by Boos4721** [reply]
> 对了，29号前的正常版本5Gwifi貌似有点问题，iphone13无法连接，（其他设备连接正常）设置为wpa和wpa2都一样，改成AC模式才成功连接

known issuse, waiting for new backport.

**Reply by Boos4721** [reply]
> 1G版本固件我测试了，结果是需要用ttl救砖 自编译和你编译的都是一样的问题：1.无法保存nvram，导致无法切换回rootfs12分区。2.eth0变成了lan eth1变成了wan，eth2/3未使用。3.无线wifi均无法启动。4.无法通过sysupgrade命令恢复到正常的固件。 [AX3600_ttl_2021-12-31_5_39_44.log](https://github.com/Boos4721/openwrt/files/7794824/AX3600_ttl_2021-12-31_5_39_44.log) 。

ok，try another way `Kernel modules > Wireless Drivers > ath11k memory profile `

**Reply by Boos4721** [reply]
ill fix it after the exam.

**Reply by davnozdu** [reply]
Solution for fixing AX Wi-Fi for apple devices.
add color option in config file.

/etc/config/wireless
config wifi-device 'radio1'
option he_bss_color '22'

**Reply by wutaijieing** [reply]
> Solution for fixing AX Wi-Fi for apple devices. add color option in config file.
> 
> /etc/config/wireless config wifi-device 'radio1' option he_bss_color '22'

Thank you

**Reply by Boos4721** [reply]
> Solution for fixing AX Wi-Fi for apple devices. add color option in config file.
> 
> /etc/config/wireless config wifi-device 'radio1' option he_bss_color '22'

revert BSS commits

**Comment by wutaijieing** [comment]
make[7]: *** No rule to make target 'arch/arm64/boot/dts/qcom/ipq8071-ax3600-1G.dtb', needed by '__build'.  Stop.

**Reply by wutaijieing** [reply]
Applying /home/taijie/Desktop/openwrt/target/linux/ipq807x/patches-5.10/900-arm64-dts-add-OpenWrt-DTS-files.patch using plaintext: 
patching file arch/arm64/boot/dts/qcom/Makefile
patch: **** malformed patch at line 25:  dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-longcheer-l8150.dtb

Patch failed!  Please fix /home/taijie/Desktop/openwrt/target/linux/ipq807x/patches-5.10/900-arm64-dts-add-OpenWrt-DTS-files.patch!
make[4]: *** [Makefile:31: /home/taijie/Desktop/openwrt/build_dir/target-aarch64_cortex-a53_musl/linux-ipq807x_generic/linux-5.10.89/.prepared_561e8e1e2f449a33af841548252e8a8c] Error 1
make[4]: Leaving directory '/home/taijie/Desktop/openwrt/target/linux/ipq807x'
make[3]: *** [Makefile:11: compile] Error 2
